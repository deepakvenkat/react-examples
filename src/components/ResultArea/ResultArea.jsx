import React from 'react';
import { InputBase } from '@material-ui/core';
import PropTypes from 'prop-types';

import { withStyles } from '@material-ui/core/styles';

const styles = {
  root: {
    height: '75px',
    width: '300px',
    '& .MuiInputBase-input': {
      '& .MuiInput-root-before': {
        height: 0,
      },
      height: '75px',
      width: '300px',
      borderRadius: 0,
      borderColor: 'red',
      border: 'solid 0.5px',
      padding: 0,
      fontSize: '32px',
      color: 'white',
      backgroundColor: '#420a12',
      textAlign: 'right',
      lineHeight: '32px',
    },
  },
};

const ResultArea = ({ classes, ...rest }) => (
  <InputBase
    classes={classes}
    {...rest}
  />
);

ResultArea.propTypes = {
  classes: PropTypes.shape({}).isRequired,
};

export default withStyles(styles)(ResultArea);
