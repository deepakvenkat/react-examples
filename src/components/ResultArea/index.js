import ResultArea from './ResultArea.jsx';

export default ResultArea;
