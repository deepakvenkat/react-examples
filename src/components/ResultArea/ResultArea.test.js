import React from 'react';
import { render } from '@testing-library/react';
import ResultArea from './ResultArea';

test('renders the resultarea', () => {
  const { asFragment } = render(
    <ResultArea />
  );
  expect(asFragment()).toMatchSnapshot();
});
