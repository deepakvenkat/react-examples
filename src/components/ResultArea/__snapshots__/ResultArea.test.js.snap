// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the resultarea 1`] = `
<DocumentFragment>
  <div
    class="MuiInputBase-root ResultArea-root-1"
  >
    <input
      class="MuiInputBase-input"
      type="text"
      value=""
    />
  </div>
</DocumentFragment>
`;
